src/modules/index.test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import EventObj from './addEvent.js';
+
+const event = new EventObj();
+event.addList = jest.fn();
+event.delEvent = jest.fn();
+describe('implementation of addList and delList testing', () => {
+  // Testing for the add list method
+  test('Add one new item to the list', () => {
+    document.body.innerHTML = '<div>'
+        + '  <ul id="list"><li></li></ul>'
+        + '</div>';
+    jest.fn();
+    const list = document.querySelectorAll('#list li');
+    expect(list).toHaveLength(1);
+  });
+  // Testing for the remove list method
+  test('Delete item from the list', () => {
+    document.body.innerHTML = '<div>'
+        + '  <ul id="list"></ul>'
+        + '</div>';
+    jest.fn();
+    const list = document.querySelectorAll('#list li');
+    expect(list).toHaveLength(0);
+  });
```

Comment:
- [ ] Great job setting up jest & using describe methods, but please notice that the tests that you write are not testing the add and the delete functions, they only testing the HTML that you wrote which is  
```JS
 document.body.innerHTML = '<div>'
        + '  <ul id="list"><li></li></ul>'
        + '</div>';
```
as mentioned in the requirements kindly try to test your add & delete function directly, if you need help or tips or any more explanations, mention me in the comments or contact me on slack

Reply:
well resolved! ✔️